### ISSUE TITLE
Support for linux/arm64/v8 

### ISSUE THREAD

**Issue by Feoner** [adamjsturge/xsshunter-go#36]
**Is your feature request related to a problem? Please describe.**
Container not support on arm64

**Describe the solution you'd like**
add support for arm64 architecture

**Additional context**
I have never built docker files but after some testing I did create an image with docker buildx build --load --no-cache --progress=plain --platform linux/arm64/v8 -t xsshunter-armv8 -f .\Dockerfile .. I have been able to create the container and hit the initial default page, and see in the docker logs that an admin user has been created but I am unable to hit the admin page.
![int](https://github.com/adamjsturge/xsshunter-go/assets/38793370/d6db00ef-423f-43fa-b81c-38947d5f6b68)
![admin404](https://github.com/adamjsturge/xsshunter-go/assets/38793370/abffcaee-3376-493e-b909-3e7359c9498b)
![user](https://github.com/adamjsturge/xsshunter-go/assets/38793370/e28c7fec-7498-477d-ba01-8475eeb7a963)
![build](https://github.com/adamjsturge/xsshunter-go/assets/38793370/aa842cc5-c435-4a12-afa8-1e62d0a4fc1e)

**Comment by adamjsturge** [comment]
I'll fully dive into this tonight if I find sometime. It's weird because I coded this on an arm so I assume that it should just work on arm but also realized that I wasn't building for it in the github actions. So let me take a look and I'll get back you hopefully soon

**Comment by adamjsturge** [comment]
The arm docker image has been pushed to dev https://hub.docker.com/repository/docker/adamjsturge/xsshunter-go/tags
it's currently building for latest, I don't think your issues is with the current arm setup, but if you wanna share your docker compose then I'd be able to help find the bug

**Comment by adamjsturge** [comment]
I'd remove `- ${DOCKER-VOLUME}src:/app/src # For development` 
as i think that is what might cause some issues.

**Comment by adamjsturge** [comment]
```yml
version: '3.9'
services:
    xsshunter-go:
        image: adamjsturge/xsshunter-go:latest
        volumes:
            - ${DOCKER-VOLUME}/db/:/app/db/
            - ${DOCKER-VOLUME}/screenshots/:/app/screenshots/
        env_file:
            - .env
```

**Comment by adamjsturge** [comment]
Gonna close this since #38 did fix what the original issue was about. I think I correctly solved the docker compose issue even this `- ${DOCKER-VOLUME}src:/app/src # For development` is written wrong without a `/` between src and the variable. But also you shouldn't use this for the image as that volume should only be used for when you're using the code base. I was able to replicated the other bug and removing any src volume also works because it can't find the html files that should be in src.

tl;dr
Added arm docker image
remove src volume and it works
